package.json

```diff
@@ -15,12 +15,12 @@
     "lint": "cross-env FORCE_COLOR=1 lerna run lint",
     "link": "lerna link",
     "test": "npm run test:unit && npm run test:component",
-    "test:component": "aw -c aw.config.js --testExt '*.comp.js'",
+    "test:component": "jest --testMatch **/test/component/**/*.comp.js",
     "test:integration:ci": "node ./test/scripts/start-puppet.js",
```

Comment:
You dont really have to do change this now, but this script should probably not be called "..-puppet.js"